cluster/gce/gci/configure-helper.sh

```diff
@@ -2578,6 +2578,15 @@ EOF
 function override-kubectl {
     echo "overriding kubectl"
     echo "export PATH=${KUBE_HOME}/bin:\$PATH" > /etc/profile.d/kube_env.sh
+    # Add ${KUBE_HOME}/bin into sudoer secure path.
+    local sudo_path=$(sudo env | grep "^PATH=")
+    if [ -n "${sudo_path}" ]; then
```

Comment:
Use double brackets instead: `[[ -n "${sudo_path}" ]]`, it's a built-in, more reliable in general.

Reply:
Done